cimpyorm/Model/Schema.py

```diff
@@ -11,8 +11,8 @@
 import json
 from argparse import Namespace
 import os
-from collections import ChainMap, Iterable, defaultdict
-
+from collections import ChainMap, defaultdict
+from typing import Iterable
```

Comment:
Looks like `typing.Iterable` was actually deprecated in favour of `collections.abc.Iterable` in 3.9? See https://docs.python.org/3.9/library/typing.html?highlight=iterable#typing.Iterable

Could just drop support for 3.8 (eol in 2024) maybe?

Reply:
Adjusted it accordingly. Good catch, i did that the other way round and went for the "typing" import. But yes, trend shows that the typing library is becoming more and more deprecated